app/scripts/migrations/048.js

```diff
@@ -92,7 +92,7 @@ function transformState (state = {}) {
   const addressBook = state.AddressBookController?.addressBook || {}
   Object.keys(addressBook).forEach((networkKey) => {
     if ((/^\d+$/ui).test(networkKey)) {
-      const chainId = `0x${networkKey.toString(16)}`
+      const chainId = `0x${parseInt(networkKey, 10).toString(16)}`
```

Comment:
I added a try/catch around this first, but I don't actually think we can make it throw due to the RegEx test on line 94.